packages/medusa/src/api-v2/admin/draft-orders/query-config.ts

```diff
@@ -0,0 +1,45 @@
+export const defaultAdminOrderRelations = []
+export const allowedAdminOrderRelations = []
+
+export const defaultAdminListOrderFields = [
+  "id",
+  "status",
+  "version",
+  "items.*",
```

Comment:
todo: *items, under the hood, it will do the necessary, here, if someone decide to use the listConfig or retrieveConfig, they will have a * in the select which does not work in most orm

Reply:
same for all other `.*`